### PR TITLE
added (Draft) to PRs

### PR DESCRIPTION
**Pull request by KJStick**

### CODE REVIEW

File: wxt_cidrbot/cmd_list.py [hunk 1]
```diff
@@ -474,25 +474,34 @@ def issues(self, target_user, room_id):
...
+                        self.webex.edit_message(self.msg_id, edit_message, self.room_of_msg)
+                    url = issue_dict[issue]['url']
+                    issue_type = issue_dict[issue]['type']
+                    issue_num = issue_dict[issue]['number']
+                    issue_color_code = issue_dict[issue]['color_code']
+                    is_draft = issue_dict[issue]['is_draft']
+
+                    if is_draft:
+                        draft_txt = "(Draft) "
+                    else:
```
Comment (ppajersk) [inline wxt_cidrbot/cmd_list.py:495]: This doesn't need an else block. Define draft_txt before line 493, and if is_draft, set draft_txt to the appropriate value

File: wxt_cidrbot/cmd_list.py [hunk 2]
```diff
@@ -504,24 +513,33 @@ def get_user_issues(self, assigned_issues_dict, user):
...
+                    issue_name = assigned_issues_dict[issue]['name']
+                    url = assigned_issues_dict[issue]['url']
+                    issue_type = assigned_issues_dict[issue]['type']
+                    issue_num = assigned_issues_dict[issue]['number']
+                    issue_color_code = assigned_issues_dict[issue]['color_code']
+                    is_draft = assigned_issues_dict[issue]['is_draft']
+
+                    if is_draft:
+                        draft_txt = "(Draft) "
+                    else:
```
Comment (ppajersk) [inline wxt_cidrbot/cmd_list.py:534]: Same here as before

File: wxt_cidrbot/git_api_handler.py [hunk 3]
```diff
@@ -125,7 +125,17 @@ def process_issue(self, issue, request, issue_type, issue_num, assign_type):
...
 
-        text = f"{issue_color_code} &nbsp; {issue_type} #{issue_num}: {hyperlink_format}"  # &nbsp; represents a space character
+        try:
+            is_draft = issue['draft']
+        except KeyError:
+            is_draft = False
+
+        if is_draft:
+            draft_txt = "(Draft) "
+        else:
```
Comment (ppajersk) [inline wxt_cidrbot/git_api_handler.py:135]: #3

**Review by ppajersk — CHANGES_REQUESTED** [review]

**Review by jlothian — CHANGES_REQUESTED** [review]

**Review by ppajersk — APPROVED** [review]
LGTM

**Review by jlothian — APPROVED** [review]

**Pull request merged**